panel/chat/feed.py

```diff
@@ -724,9 +726,15 @@ def add_step(
         avatar : str | bytes | BytesIO | None
             The avatar to use; overrides the message's avatar if provided.
             Will default to the avatar parameter. Only applicable if steps is "new".
-        steps_column : Column | None
+        steps_layout : Column | None
             An existing Column of steps to stream to, if None is provided
```

Comment:
As its now a layout, I believe the text explanation needs to change from Column to something more general.